### PR TITLE
1843-suppressedHoldingLocations

### LINKED ISSUE #1843: Skip Alma holdings/items at suppressed location.
Alma allows to suppress locations of holdings/items, but this fact is _currently_ not reflected in the publishing (see Ex Libris Support Case #06393609: "The development team plans to redesign and enhance the functionality to include the holdings suppressed on the location level to be suppressed for the General publishing [...], but we cannot commit to a timeline at this point."). Any holdings/items (as well as any derived information) whose current location (`H52$$c`/`ITM$$x`) is suppressed need to be skipped.

Examples:

- [HT015014677](https://lobid.org/resources/990156060190206441) (DE-Bi10):

```xml
<datafield tag="H52" ind1="8" ind2=" ">
  <subfield code="b">40</subfield>
  <subfield code="c">CB_Abschl</subfield>
  <subfield code="h">MA80 Wien</subfield>
  <subfield code="8">2221203190006450</subfield>
</datafield>
```

- [HT000909138](https://lobid.org/resources/990058567920206441) (DE-708):

```xml
<datafield tag="ITM" ind1=" " ind2=" ">
  <subfield code="H">22110796420006464</subfield>
  <subfield code="x">AUSTRA</subfield>
  <subfield code="f">BOOK</subfield>
  <subfield code="v">30</subfield>
  <subfield code="p">82</subfield>
  <subfield code="X">USP000332308</subfield>
  <subfield code="U">1998-07-07 11:22:44 Europe/Berlin</subfield>
  <subfield code="Y">2022-10-14 11:22:45 Europe/Berlin</subfield>
  <subfield code="h">false</subfield>
  <subfield code="n">ODU/DEM</subfield>
  <subfield code="M">49HBZ_FUH</subfield>
  <subfield code="s">1</subfield>
  <subfield code="d">#</subfield>
  <subfield code="V">import</subfield>
  <subfield code="b">801553101</subfield>
  <subfield code="a">23110796410006464</subfield>
  <subfield code="B">80/15531</subfield>
  <subfield code="D">2000-02-17 10:22:44</subfield>
  <subfield code="W">1998-07-07 02:00:00 Europe/Berlin</subfield>
  <subfield code="u">F0001</subfield>
  <subfield code="w">F0001</subfield>
</datafield>
```

The mapping of suppressed locations should be shared with DigiBib IntrOX ([Limetrans](https://github.com/search?q=repo%3Ahbz%2Flimetrans+suppressedLocation+path%3A%2F%5Esrc%5C%2Fconf%5C%2Fintrox%5C%2F%2F&type=code)) to ensure consistency and reduce maintenance burden. Ideally, it would be updated regularly based on the Alma API (each library that wants their suppressed locations being taken into account would need to provide an API key for the [Configuration API](https://developers.exlibrisgroup.com/alma/apis/conf/#Organization%20Units): Area = Configuration, Env = Production, Permissions = Read-only).

### PR DESCRIPTION
**Pull request by TobiasNx**
Unfortunatly I have no test for H52. The test stated in https://github.com/hbz/lobid-resources/issues/1843 is not relevant anymore. @blackwinter do you have another test for me.

Resolves #1843

### CODE REVIEW

File: web/conf/context.jsonld
Comment (blackwinter) [inline web/conf/context.jsonld:1]: Does this change belong here?

**Comment by TobiasNx** [comment]
Yes HOL without ITM but with H52.

**Comment by blackwinter** [comment]
I meant "work" as in "is it a suitable example".

**Comment by TobiasNx** [comment]
I tested it with your example, saw that we already have a suitable example but my transformation selected an not existing subfield `H52 $M`. This should be fixed now.

**Review by dr0i — APPROVED** [review]

File: web/conf/context.jsonld
Comment (TobiasNx) [reply web/conf/context.jsonld:1]: this was an error. but lets keep it in. Next time I only commit if the context changes. Also see https://github.com/hbz/lobid-resources/pull/1875

**Review by dr0i — APPROVED** [review]

**Comment by dr0i** [comment]
> Too bad I didn't get the chance to approve this pull request...

ups, sorry, I thought we were good here ... it's not deployed yet, shall I revert ?

**Review by blackwinter — COMMENTED** [review]
:+1:

**Comment by dr0i** [comment]
Ok, I let it as it is.
To be honest I was first hesitant if I should merge it. Note to self: better ask two times too much than one time too little.

**Comment by dr0i** [comment]
Will be deployed on Monday.

**Pull request merged**